osutil/user.go

```diff
@@ -46,12 +55,16 @@ func AddExtraSudoUser(name string, sshKeys []string, gecos string) error {
 		"--gecos", gecos,
 		"--extrausers",
 		"--disabled-password",
-		"--add_extra_groups", "sudo",
 		name)
 	if output, err := cmd.CombinedOutput(); err != nil {
 		return fmt.Errorf("adduser failed with %s: %s", err, output)
 	}
 
+	sudoersFile := filepath.Join(sudoersDotD, "create-user-"+name)
```

Comment:
Ah good point. ~ is not permitted anyway but . is. I could replace . with some other character but it would have to not be one of the permitted characters or we could get collisions. : or , or something I guess...